packages/tupaia-web-server/src/routes/SubscribeRoute.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { TupaiaWebSubscribeRequest } from '@tupaia/types';
+
+export type SubscribeRequest = Request<
+  TupaiaWebSubscribeRequest.Params,
+  TupaiaWebSubscribeRequest.ResBody,
+  TupaiaWebSubscribeRequest.ReqBody,
+  TupaiaWebSubscribeRequest.ReqQuery
+>;
+
+export class SubscribeRoute extends Route<SubscribeRequest> {
+  public async buildResponse() {
+    const {
+      ctx,
+      params: { projectCode, entityCode, dashboardCode },
+    } = this.req;
+
+    const dashboard = await this.req.models.dashboard.findOne({ code: dashboardCode });
+    const project = await this.req.models.project.findOne({ code: projectCode });
+    const entity = await this.req.models.entity.findOne({ code: entityCode });
+
+    const dashboardMailingList = await this.req.models.dashboardMailingList.findOne({
+      project_id: project.id,
+      entity_id: entity.id,
+      dashboard_id: dashboard.id,
+    });
```

Comment:
Could we use central-server to fetch these rather than directly pulling from the db?
`await this.req.ctx.services.central.fetchResources(...)`